Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Domain/Model/DbContextBase.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.IO;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+
+namespace MvvmSeed.Domain.Model
+{
+    public abstract class DbContextBase : DbContext
```

Comment:
Something is bugging me about the responsibility here that I'm not sure if is right or wrong. This class looks like a base, something generic but it also has some sqlite-specific code. Again, not sure... but you are the single responsibility principle dude.